manifests/charts/istiod-remote/templates/mutatingwebhook.yaml

```diff
@@ -0,0 +1,66 @@
+# Installed for each revision - not installed for cluster resources ( cluster roles, bindings, crds)
+{{- if not .Values.global.operatorManageWebhooks }}
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+{{- if eq .Release.Namespace "istio-system"}}
+  name: istio-sidecar-injector{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
+{{ else }}
+  name: istio-sidecar-injector{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}-{{ .Release.Namespace }}
+{{- end }}
+  labels:
+    istio.io/rev: {{ .Values.revision | default "default" }}
+    app: sidecar-injector
+    release: {{ .Release.Name }}
+webhooks:
+  - name: sidecar-injector.istio.io
+    clientConfig:
+      url: https://{{ .Values.sidecarInjectorWebhook.remoteWebhookHost }}:{{ .Values.sidecarInjectorWebhook.remoteWebhookPort }}/inject
+      caBundle: ""
```

Comment:
Need to figure out how to retrieve the CA bundle to validate the webhook's server certificate.

Istiod will patch the `MutatingWebhookConfiguration` with CA bundle before starting if `MutatingWebhookConfiguration` is in the same cluster.

How to patch it if the `MutatingWebhookConfiguration` in different cluster, will be some manual steps needed?

Reply:
Plan to modify the WebHooks code to patch it when centralIstiod is enabled.